flaskr/__init__.py

```diff
@@ -94,7 +94,7 @@ def results():
 		else:
 			return redirect(url_for('index'))
 		if 'code' in session:
```

Comment:
I want to add @khzheng24998's commit from #34 here after resolving merge conflicts